modules/custom/movie_reservation/templates/movie_resertvation.html.twig

```diff
@@ -0,0 +1,8 @@
+<p> All Movies </p>
```

Comment:
Zašto imaš dva twiga?

Reply:
Greskom nisam stage-ovala prethodno obrisani twig, pa je doslo do greske. Sad sam ispravila to.